Makefile

```diff
@@ -32,11 +32,13 @@ build-frontend: init-frontend
 	npm run build
 
 .PHONY: tests-frontend
-tests-frontend: init-frontend
+tests: tests-frontend
+tests-frontend:
 	@cd ${RECIPLEASE_WEB_PATH} && \
-	npm test
+	${YARN} test
```

Comment:
what's `${YARN}`

Reply:
In Make, it is common practice to call dependencies using environment variables. In your `config/.env.local` file you would have a line `YARN=yarn`. This is done to make the code platform-agnostic.